<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registration</class>
 <widget class="QWidget" name="Registration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="layout_directory">
     <item>
      <widget class="QLabel" name="directory_label">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>29</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>29</height>
        </size>
       </property>
       <property name="text">
        <string>Directory</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="directory_edit">
       <property name="minimumSize">
        <size>
         <width>256</width>
         <height>29</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>256</width>
         <height>29</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="directory_browse_button">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="directory_help">
       <property name="minimumSize">
        <size>
         <width>23</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>?</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layout_password">
     <item>
      <widget class="QLabel" name="password_label">
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>29</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>61</width>
         <height>29</height>
        </size>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="password_edit">
       <property name="minimumSize">
        <size>
         <width>256</width>
         <height>29</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>256</width>
         <height>29</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="password_help">
       <property name="minimumSize">
        <size>
         <width>23</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>?</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layout_strength">
     <item>
      <widget class="QLabel" name="strength_label">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>46</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>61</width>
         <height>46</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Password&lt;/p&gt;&lt;p&gt;Strength&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="strength_progress_bar">
       <property name="minimumSize">
        <size>
         <width>253</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>253</width>
         <height>25</height>
        </size>
       </property>
       <property name="value">
        <number>24</number>
       </property>
       <property name="format">
        <string>%p%</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="strength_help_button">
       <property name="minimumSize">
        <size>
         <width>23</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>?</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="expert_checkBox">
     <property name="text">
      <string>Expert Options</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layout_secret_file">
     <item>
      <widget class="QLabel" name="secret_label">
       <property name="minimumSize">
        <size>
         <width>68</width>
         <height>29</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>68</width>
         <height>29</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Secret File&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="secret_edit">
       <property name="minimumSize">
        <size>
         <width>256</width>
         <height>29</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>256</width>
         <height>29</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="secret_button">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="secret_help_button">
       <property name="minimumSize">
        <size>
         <width>23</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>?</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layout_ok_and_cancel">
     <item>
      <widget class="QPushButton" name="cancel_button">
       <property name="minimumSize">
        <size>
         <width>83</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>83</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ok_button">
       <property name="minimumSize">
        <size>
         <width>83</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>83</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
